doc/contributing.rst

```diff
@@ -253,42 +253,98 @@ How to make a release
 
 To make a new release of the package, follow these steps:
 
-1. Check that the nightly build on CircleCI was successful
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Check the tests on GitHub Actions and CircleCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Check the ``nightly`` `build on
-CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__.
-All tests should pass before making a release.
+Check the ``nightly``
+`build on CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__
+and the
+`GitHub Actions run <https://github.com/ESMValGroup/ESMValCore/actions>`__.
+All tests should pass before making a release (branch).
 
-2. Make a pull request to increase the version number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2. Increase the version number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The version number is stored in ``esmvalcore/_version.py``,
-``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files. See
-https://semver.org for more information on choosing a version number.
-
-3. Make the release on GitHub
+``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files.
+Also update the release date in ``CITATION.cff``.
+See https://semver.org for more information on choosing a version number.
+Make a pull request and get it merged into ``master``.
+
+3. Add release notes
+~~~~~~~~~~~~~~~~~~~~
+Use the script
+`esmvaltool/utils/draft_release_notes.py <https://docs.esmvaltool.org/en/latest/utils.html#draft-release-notes-py>`__
+to create create a draft of the release notes.
```

Comment:
probably best to add that we need `pygithub` and can be installed by running `pip install pygithub` and a personal token obtainable by following instructions at https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line (I know the script is verbose and tells you that, but let's have the release manager have the tools installed already and not see it fail the first time)

Reply:
I've improved the documentation here: https://github.com/ESMValGroup/ESMValTool/pull/1848

Reply:
yah that's nice, just took a look, good one, man!